app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
+
+    def die(self) -> None:
+        self.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    def bite(self, herbivore: Herbivore) -> None:
```

Comment:
it's staticmethod because you don't use self in this method and  here I'd had the same problem with annotation
![image](https://github.com/mate-academy/py-herbivores-and-carnivores/assets/37306964/09d7c62e-f584-4d4c-ab77-99fa82981ba4)

Reply:
Done